src/pages/Stream/index.tsx

```diff
@@ -58,6 +61,16 @@ const Stream: FC = () => {
 		isRefetching: isSchemaRefetching,
 	} = useGetStreamSchema({ streamName: currentStream || '' });
 
+	useEffect(() => {
+		setLogsStore((store) => {
+			if (!timeRangeInitialised) {
```

Comment:
Why does this exit statement exist here?
Move this to the appropriate place.

Reply:
Addressed this.